### PR TITLE
Implement [ACR] Rooftop Bypass

### PR DESCRIPTION
**Pull request by grimreap124**
https://scryfall.com/card/acr/298/rooftop-bypass

### CODE REVIEW

File: Mage.Sets/src/mage/cards/r/RooftopBypass.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * @author grimreap124
+ */
+public final class RooftopBypass extends CardImpl {
+
+    public RooftopBypass(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{U}{B}");
+        
+
+        // Whenever one or more nontoken creatures you control deal combat damage to a player, create a 1/1 black Assassin creature token with menace.
+        this.addAbility(new DealCombatDamageControlledTriggeredAbility(new CreateTokenEffect(new AssassinToken3()),
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/r/RooftopBypass.java:24]: we already have `AssassinMenaceToken`

Comment (theelk801) [reply Mage.Sets/src/mage/cards/r/RooftopBypass.java:24]: also you need to pass a filter into the triggered ability

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/r/RooftopBypass.java (hunk 1 above)
Comment (grimreap124) [reply Mage.Sets/src/mage/cards/r/RooftopBypass.java:24]: I'm confused as to what you mean by passing a filter? I believe I am StaticFilters.FILTER_CONTROLLED_CREATURES_NON_TOKEN?

Comment (theelk801) [reply Mage.Sets/src/mage/cards/r/RooftopBypass.java]: yeah I just somehow missed that the filter was on the next line, you're good

**Review by theelk801 — APPROVED** [review]

**Pull request merged**